app/assets/v2/css/base.css

```diff
@@ -294,7 +294,7 @@ p a {
 }
 
 .btn-warning:hover, .btn-success:hover, .btn-info {
```

Comment:
can put `.btn` style here, and it will be better to put them in separate line

Reply:
Agreed.  This is existing code that wasn't modified, just fixed the merge conflict.

The scss is being fixed by @thelostone-mc in another branch.